dependencies/server-dependencies/pom.xml

```diff
@@ -73,7 +73,7 @@
             in JBoss -->
         <dependency>
             <groupId>org.hibernate.javax.persistence</groupId>
-            <artifactId>hibernate-jpa-2.0-api</artifactId>
+            <artifactId>hibernate-jpa-2.1-api</artifactId>
```

Comment:
I've rechecked and EAP 6.3 and EAP 6.4 bundles Hibernate 4.2.17+, which is JPA 2.0.

The fact that it deploys onto EAP 6.4 without issues with JPA 2.1 orm.xml descriptor is irrelevant :-)

Here is an "official" statement: https://access.redhat.com/solutions/404223

----

We basically need JPA 2.1 only for describing index on `installation#deviceToken` column. We have alternatives here:

1. go just with index (primary key) on `installation#id` column (as we have it now), then we would need to order by id column, which means we get rid of `distinct` in query (and de-duplicate device tokens in result collection)